doc/multipath.md

```diff
@@ -63,7 +63,7 @@ matching by position and type.
   - Moreover, drives that were assigned to such disks will be ignored later.
   - **Pitfall:** a disk with `bios_id=0x80` can match a `CT_DMMULTIPATH` drive.
     That messes everything up.
-  
+
```

Comment:
what about this file? Should it be here or in storage-ng?

Reply:
This document is about the old AutoYaST+storage code, which used to live in `autoyast2`. For that reason, we are keeping it here.